tests/test_retrieve.py

```diff
@@ -0,0 +1,114 @@
+import pytest
+from helper import create_two, register, PASSWORD, send, scmail, create_key
+
+
+"""
+Handle cases of retrieve:
+
+- parameter checking
+    - two fingerprint
+
+- no message
+- one messages
+- multiple messages.
+-
+"""
+
+
+@pytest.mark.finished
+def test_no_sender(caplog, runner):
+    fingerprint, sender = create_two(runner)
```

Comment:
I realize it's not a problem since this just returns two fingerprints but `create_two`'s return statement looks like this which is not consistent with how it's destructured here.
```python
    ...
    return sender_fingerprint, recipient_fingerprint
```

Reply:
Thanks, I fix this.